O365/utils/utils.py

```diff
@@ -321,15 +321,37 @@ def __init__(self, attribute=None, *, protocol):
         self._filters = []  # store all the filters
         self._order_by = OrderedDict()
         self._selects = set()
+        self._expands = set()
 
     def __str__(self):
-        return 'Filter: {}\nOrder: {}\nSelect: {}'.format(self.get_filters(),
-                                                          self.get_order(),
-                                                          self.get_selects())
+        return 'Filter: {}\nOrder: {}\nSelect: {}\nExpand: {}'.format(self.get_filters(),
+                                                                      self.get_order(),
+                                                                      self.get_selects(),
+                                                                      self.get_expands())
 
     def __repr__(self):
         return self.__str__()
 
+    @fluent
+    def expand(self, *relationships):
+        """ Adds the relationships (e.g. "microsoft.graph.eventMessage/event" or
+        "attachments") that should be expanded to the $expand parameter
+
+        :param str relationships: the relationships tuple to expand.
+        :rtype: Query
+        """
+
+        for relationship in relationships:
```

Comment:
To control whether to add 'microsoft.graph.eventMessage' if the protocol is MSGrapProtocol or the 'Microsoft.OutlookServices.EventMessage' if you are using Outlook rest api, you have to call self.protocol.get_service_keyword. I'll implement this